src/objects/_main-wrapper.scss

```diff
@@ -2,52 +2,27 @@
 @import "../tools/all";
 @import "../helpers/all";
 
-// Example usage with Breadcrumbs, phase banners, back links:
-// <div class="govuk-width-container">
-//   <!-- Breadcrumbs, phase banners, back links are placed in here. -->
-//   <div class="govuk-main-wrapper">
-//       <!-- Wrapper for the main content of your page which applies padding
-//            to the top / bottom -->
-//   </div>
-// </div>
-//
-// Example usage without Breadcrumbs, phase banners, back links:
-// <div class="govuk-width-container">
-//   <div class="govuk-main-wrapper govuk-main-wrapper--l">
-//       <!-- Wrapper for the main content of your page which applies padding
-//            to the top / bottom -->
-//   </div>
-// </div>
-
-
-@mixin govuk-main-wrapper {
-  // In IE11 the `main` element can be used, but is not recognized  –
-  // meaning it's not defined in IE's default style sheet,
-  // so it uses CSS initial value, which is inline.
-  display: block;
-  padding-top: govuk-spacing(4);
-  padding-bottom: govuk-spacing(4);
-
-  @include govuk-media-query($from: tablet) {
-    // This spacing is manually adjusted to replicate the margin of
-    // govuk-heading-xl (50px) minus the spacing of back link and
-    // breadcrumbs (10px)
-    padding-top: govuk-spacing(7);
-    padding-bottom: govuk-spacing(7);
-  }
-}
-
-// Use govuk-main-wrapper--l when you page does not have Breadcrumbs, phase banners or back links
-@mixin govuk-main-wrapper--l {
-  @include govuk-responsive-padding(8, "top");
-}
-
 @include govuk-exports("govuk/objects/main-wrapper") {
   .govuk-main-wrapper {
-    @include govuk-main-wrapper;
-  }
+    // In IE11 the `main` element can be used, but is not recognized  –
+    // meaning it's not defined in IE's default style sheet,
+    // so it uses CSS initial value, which is inline.
+    display: block;
+    padding-top: govuk-spacing(4);
+    padding-bottom: govuk-spacing(4);
+
+    @include govuk-media-query($from: tablet) {
+      // This spacing is manually adjusted to replicate the margin of
+      // govuk-heading-xl (50px) minus the spacing of back link and
+      // breadcrumbs (10px)
+      padding-top: govuk-spacing(7);
+      padding-bottom: govuk-spacing(7);
+    }
 
-  .govuk-main-wrapper--l {
```

Comment:
We could in theory leave this modifier around, there might be cases where it could be useful?

My reasoning for removing it is, that this modifier should only be applied when the main wrapper is the first child. cc @dashouse is this correct?

Reply:
I guess there might be edge cases where for _reasons_ a service ends up with an empty or non-visible element before the main wrapper? For example, a script tag, or an empty container that's always there even if there's no phase banner / breadcrumbs / whatever.

I _think_ I'd be tempted to keep the class just so that there's an option available in those cases?

Reply:
Based on existing implementations of Alerts (https://github.com/alphagov/govuk-design-system-backlog/issues/2) I would second keeping the class

Reply:
Makes sense to me lets keep it.